exporter/stackdriver/trace.go

```diff
@@ -116,8 +116,12 @@ func (e *traceExporter) uploadToStackdriver(spans []*trace.SpanData) {
 	for _, span := range spans {
 		req.Spans = append(req.Spans, protoFromSpanData(span, e.projectID))
 	}
-	err := e.client.BatchWriteSpans(context.Background(), &req)
+	// create a never-sampled span to prevent traces associated with exporter
+	span := trace.NewSpan("ignored: uploadToStackdriver", nil, trace.StartOptions{Sampler: trace.NeverSample()})
+	ctx := trace.WithSpan(context.Background(), span) // TODO: add timeouts
+	err := e.client.BatchWriteSpans(ctx, &req)
```

Comment:
Not sure if this can happen (I cannot understand based on the documentation), but if 0 spans are exported we should just immediately return. So I suggest to check that as the first thing in the uploadToStackdriver

Reply:
I don't think this is possible